keyboards/rmi_kb/tkl_ff/info.json

```
@@ -1,20 +1,12 @@
 {
     "keyboard_name": "TKL FF",
-    "manufacturer": "Ramon Imbao",
+    "manufacturer": "RMI-KB",
     "url": "",
-    "maintainer": "Ramon Imbao",
+    "maintainer": "ramonimbao",
     "usb": {
         "vid": "0xB16B",
         "device_version": "0.0.1"
     },
```

Not sure why this was removed
```suggestion
    },
    "layout_aliases": {
        "LAYOUT_ansi": "LAYOUT_tkl_f13_ansi",
        "LAYOUT_iso": "LAYOUT_tkl_f13_iso"
    },
    "community_layouts": [
        "tkl_f13_ansi",
        "tkl_f13_iso"
    ],
```